.github/workflows/release_stable.yaml

```diff
@@ -31,6 +31,8 @@ jobs:
           else
             echo "needs_releasing=false" >> $GITHUB_OUTPUT
           fi
+      - name: Verify image ID on mainnet chains
```

Comment:
We need to skip, not make it red

Reply:
make `release` red, don't make `check-if-needs-releasing` red

Reply:
done